src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+p {
```

Comment:
You can use the BEM modifiers here, creating a specific modifier to `<p >` tag without margin. Like:

```
.p {
   // ...
   &--no-margin {
      margin: 0;
   }
}
```